### ISSUE TITLE
fix new repo topic creation and reenable topic asserts

### ISSUE THREAD

**Issue by claymccoy** [atomist/sdm-smoke-test#9]

**Comment by claymccoy** [comment]
The problem is that OnFirstPushToRepo fires after the new repo is created, and it has actions to add the topics to the new repos, but there is no "screenName" in the event.
https://github.com/atomist/sdm/blob/2dc5ef52b627f477258a777fbcb9db671d60daf5/src/handlers/events/repo/OnFirstPushToRepo.ts#L53
```
 "after": {
          "committer": {
            "login": "atomist-bot",
            "person": null
          },
```
So the handler just warns that it can't find the screen name and returns success. I'll investigate why there is no person data for the event, but requiring a screen name in order to invoke actions like adding topics seems error prone. I am missing some intent here, but perhaps this should be divided into two types of actions, ones that require screennames, and channels, and ones that don't that will get executed anyway. Or perhaps we just send empty addressChannels and let the actions do whatever they can.

**Comment by claymccoy** [comment]
fixed by atomist/sdm#388
